public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -204,9 +208,11 @@ export class PanelInspector extends PureComponent<Props, State> {
     }
     if (error.data) {
       return (
-        <div>
+        <div style={{ overflowY: 'auto', height: '100%' }}>
```

Comment:
Uhm, what about wrapping every but Table content with CustomScrollbar for consistency?